### DISCUSSION TITLE
Swipe Action To Return From Thread

### DISCUSSION THREAD

**Discussion by timmgleason** [tuskyapp/Tusky#3036] [Ideas]
Tapping/clicking a thread slides left to enter into the thread. This is easy to do when using a single hand. Returning to the feed currently requires a tap/click the arrow in the upper left. Difficult to do when using right hand alone. 
A swipe action added to the UI would make single handed operation easier.

**Comment by charlag** [comment]
There's already back button/back gesture from OS, do we really need another one?

**Reply by timmgleason** [reply]
The app already has swipe actions as part of it's controls (changing between Tabs).  On my Android device, both the back button and the back arrow are in the far left of the screen. Using the app one handed with the right hand puts both of those buttons on the extent of reach. One completely out of reach and the other uncomfortably in reach.
Not all hands are built the same, and some people don't have the option of two handed operation. In terms of accessibility, more options is typically better.

**Reply by AkazaRenn** [reply]
> The app already has swipe actions as part of it's controls (changing between Tabs).  On my Android device, both the back button and the back arrow are in the far left of the screen. Using the app one handed with the right hand puts both of those buttons on the extent of reach. One completely out of reach and the other uncomfortably in reach.
> Not all hands are built the same, and some people don't have the option of two handed operation. In terms of accessibility, more options is typically better.

Android has back gesture that allows you to go back by swiping inwards from EITHER side of the screen

**Reply by AkazaRenn** [reply]
> >  swiping inwards from EITHER side of the screen
> 
> @AkazaRenn whats your phone model and OS version? Because on my device (Xiaomi RN10Pro) swipe left takes me back one site/screen, but wipe from right side does nothing.

Sony Xperia 5 IV. I don't know much about those heavily modded versions of Android but it's definitely a thing for near-stock Android from Google/Sony/Motorola.
